serialization-avro/src/main/java/io/axoniq/dev/samples/serializationavro/command/GiftCardException.java

```diff
@@ -0,0 +1,57 @@
+package io.axoniq.dev.samples.serializationavro.command;
+
+
+import io.axoniq.dev.samples.serializationavro.api.GiftCardBusinessErrorCode;
+
+/**
+ * Domain specific exception
+ *
+ * @author Stefan Andjelkovic
+ */
+class GiftCardException extends RuntimeException {
```

Comment:
If we drop the `ExceptionWrappingHandlerInterceptor`, I think we can drop this as well.